job-dsls/jobs/prodTag_pipeline.groovy

```diff
@@ -4,7 +4,7 @@ def kieVersion=Constants.KIE_PREFIX
 def baseBranch=Constants.BRANCH
 def organization=Constants.GITHUB_ORG_UNIT
 def m2Dir = Constants.LOCAL_MVN_REP
-def TPB="target product build"
+def TPB="RHPAM_"
```

Comment:
what happen if the default is not changed? Does it work just with 'RHPAM_' ?

Reply:
fixed - now the entry is required. If it is left empty the build is aborted. (Before doing anything)